Apps/.editorconfig

```diff
@@ -649,3 +649,9 @@ dotnet_diagnostic.CA5403.severity = warning
 
 # CA9999: Analyzer version mismatch
 dotnet_diagnostic.CA9999.severity = warning
+
+# CS8604: Possible null reference argument.
+dotnet_diagnostic.CS8604.severity = silent
```

Comment:
Not sure we want this. It would hide a lot of valid warnings, same with the line below